### PR TITLE
Change MD5 to SHA256 fingerprint for new github.com fingerprint

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Change MD5 to SHA256 fingerprint for new github.com fingerprint

### CODE REVIEW

**Comment by krionbsd** [comment]
re-run full all

File: tests/integration/modules/test_ssh.py [hunk 1]
```diff
@@ -132,7 +132,9 @@ def test_recv_known_host_entries(self):
         """
         Check that known host information is returned from remote host
         """
-        ret = self.run_function("ssh.recv_known_host_entries", ["github.com"])
+        ret = self.run_function(
+            "ssh.recv_known_host_entries", ["github.com"], **dict(enc="ssh-rsa")
```
Comment (waynew) [inline tests/integration/modules/test_ssh.py:136]: **question** any reason to use dict unpacking here, instead of just `enc="ssh-rsa"`?

**Review by waynew — CHANGES_REQUESTED** [review]
I *think* this looks good, but one question

**Comment by waynew** [comment]
Used `ssh-keyscan github.com` to verify that the keys were accurate :+1:

File: tests/integration/modules/test_ssh.py (hunk 1 above)
Comment (krionbsd) [reply tests/integration/modules/test_ssh.py]: Thanks, fixed, unpacking was just the first thing which came into my mind

**Review by s0undt3ch — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]

**Pull request merged**